packages/core/content-type-builder/server/controllers/validation/__tests__/component.test.js

```diff
@@ -28,6 +28,7 @@ describe('Component validator', () => {
         component: {
           category: 'default',
           displayName: 'mycompo',
+          icon: 'Calendar',
```

Comment:
These should be lowercase too, no? Why do they not fail?

Reply:
I agree, but I think it would be good to validate the text is lowercase, because that is basically user-generated text from the schema that can be changed manually and the stricter we are the better :)